### PR TITLE
Added Checkers

### PR DESCRIPTION
**Pull request by Capnsockless**
Added a 8x8 checkerboard, uses the chessboard sprites and the generic sprites from parchis for the pieces. The extra "queen" pieces are just the black and white pieces with a yellow crown on them.

## About the PR 

**Screenshots**

Screenshot_3

**Changelog**

:cl:
- add: Added a checkerboard.

### CODE REVIEW

File: Content.Server/Tabletop/TabletopCheckerSetup.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+            // Queens
+            for (int i = 2; i < 5; i++)
+            {
+                EntityUid tempQualifier = entityManager.SpawnEntity("BlackQueen", new MapCoordinates(x + 9 * separation + 9f / 32, y - i * separation, mapId));
+                session.Entities.Add(tempQualifier);
+
+                EntityUid tempQualifier1 = entityManager.SpawnEntity("WhiteQueen", new MapCoordinates(x + 8 * separation + 9f / 32, y - i * separation, mapId));
+                session.Entities.Add(tempQualifier1);
+            }
```
Comment (mirrorcult) [inline Content.Server/Tabletop/TabletopCheckerSetup.cs:45]: arent these usually called kings in checkers

File: Content.Server/Tabletop/TabletopCheckerSetup.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        }
+
+        // TODO: refactor to load FEN instead
```
Comment (mirrorcult) [inline Content.Server/Tabletop/TabletopCheckerSetup.cs:49]: guessing this is c+p artifact from chess

File: Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard.rsi
+    state: chessboard
+  - type: TabletopGame
+    boardName: tabletop-checker-board-name
+    size: 338, 274
+    setup: !type:TabletopCheckerSetup
+
+# Chessboard tabletop item (item only visible in tabletop game)
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:16]:
```suggestion
# Checkerboard tabletop item (item only visible in tabletop game)
```

File: Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- type: entity
+  id: CheckerBoardTabletop
+  name: checkerboard
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard_tabletop.rsi
+    state: chessboard_tabletop
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:24]: would be nice if it used a unique sprite but i suppose checkerboards are sometimes the same as chessboards

File: Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  - type: Sprite
+    netsync: false
+    noRot: true
+  - type: Appearance
+    visuals:
+    - type: TabletopItemVisualizer
+
+# The pieces
+- type: entity
+  id: WhitePiece
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:43]: should really be called `WhiteCheckerPiece`, same for the rest

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
color contrast on the white queen is a little off

**Review by mirrorcult — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  parent: BaseItem
+  id: CheckerBoard
+  name: checkerboard
+  description: A checkerboard. Pieces included!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Tabletop/chessboard.rsi
+    state: chessboard
+  - type: TabletopGame
+    boardName: tabletop-checker-board-name
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:12]: this isnt properly localized

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml (hunk 6 above)
Comment (Capnsockless) [reply Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:12]: What does that mean?

Comment (MeltedPixel) [reply Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml:12]: Mirrorcult is saying, you didn't properly localize what tabletop-checker-board-name is. You should add the correct localization and get it from tabletop.ftl. Look at home the other board games do it.

**Review by mirrorcult — APPROVED** [review]

**Comment by mirrorcult** [comment]
i think theres a tabletop game crate or something, should add it there

**Pull request merged**